### ISSUE TITLE
[Feature Request]: Is there an option to use old workflow menu?

### ISSUE THREAD

**Issue by Lalimec** [Comfy-Org/ComfyUI_frontend#1267]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

Workflow tab at the sidebar is really uncomfortable to use, i need to make it wider to properly read the names of the wfs due to the weird padding on the left. As a result it takes a lot of space. Yes, its not there all the time but i always keep the que (quite buggy on its own) open and i have to resize it each time. I dont want to use the older version of the ui (--front-end-version Comfy-Org/ComfyUI_frontend@1.2.64), it causes tons of other issues like images not being loaded, ui becoming unresponsive, broken node packs etc. 

### Proposed workflow

![image](https://github.com/user-attachments/assets/5507ba14-b774-4cdd-bbbc-80e4b978f20f)
Making this a floating dockable module might work, i dont know. Maybe people are loving the new version.

**Comment by JorgeR81** [comment]
> Workflow tab at the sidebar is really uncomfortable to use, i need to make it wider to properly read the names of the wfs

> i have to resize it each time

This would not happen if the sidebar width, set by the user, as stored, per tab.

I asked about this, along with other sidebar improvements  ( see here:  https://github.com/Comfy-Org/ComfyUI_frontend/issues/897 ).

But maybe I should've opened multiple issues instead ...

**Comment by Lalimec** [comment]
Yeah that makes sense. Smaller font and compact designs are all good suggestions. Still, workflow related functions are all scattered around ui, and they are in different forms, tab, menu, button etc. Sure, one can gen used to it, but why lol.

**Comment by LukeG89** [comment]
> But maybe I should've opened multiple issues instead ...

Yes @JorgeR81, you should split the requests, there are more chances to make them happen (in particular I really want the stored sidebar width)

> Still, workflow related functions are all scattered around ui, and they are in different forms, tab, menu, button etc. Sure, one can gen used to it, but why lol.

@Lalimec Other users complained about that too, but I think they made it that way to be more like other softwares (dropdown menus, custom keybinds, etc.). For example, now I use more shortcuts and less clicking.

**Comment by yoland68** [comment]
You can set the tab to top bar if your issue is that it's hard to navigate around.

<img width="1063" alt="Screenshot 2024-10-17 at 10 17 13 AM" src="https://github.com/user-attachments/assets/d9d74e69-648f-45d8-9d53-52c5a6d1f8a1">

**Comment by Lalimec** [comment]
> You can set the tab to top bar if your issue is that it's hard to navigate around.
> 
> <img alt="Screenshot 2024-10-17 at 10 17 13 AM" width="1063" src="https://private-user-images.githubusercontent.com/4950057/377567905-d9d74e69-648f-45d8-9d53-52c5a6d1f8a1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkxODY3OTIsIm5iZiI6MTcyOTE4NjQ5MiwicGF0aCI6Ii80OTUwMDU3LzM3NzU2NzkwNS1kOWQ3NGU2OS02NDhmLTQ1ZDgtOWQ1My01MmM1YTZkMWY4YTEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAxNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMTdUMTczNDUyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWM4ZDQzMWUwNWI1ODQ0MmQ2NDg3NWRmNDVkOWZmNGI1NTU5ZDZiZGJkMWUxZTU3MDVmMGJiZjFmOWI0OTQ2NiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.mAt_BEnhQCb60CEGAN6Ue1j8s9jIo9ZjGQztzJjoTTc">

Thats a different thing though, its for opened tabs.
